packages/x-charts/src/RadarChart/RadarDataProvider/radar.types.ts

```diff
@@ -0,0 +1,33 @@
+export interface MetricConfig {
+  /**
+   * The name of the metric.
+   */
+  name: string;
+  /**
+   * The minimum value of the domain.
+   * @default 0
+   */
+  min?: number;
+  /**
+   * The maximal value of the domain.
+   * If not provided, it gets computed to display the entire chart data.
+   */
+  max?: number;
+}
+
+export interface RadarConfig {
+  /**
+   * The metrics shown by radar.
+   */
+  metrics: string[] | MetricConfig[];
```

Comment:
From the code, it seems we actually support `Array<string | MetricConfig>` because we're handling each metric separately:

```tsx
radar.metrics.map((m) => {
        const { name, min = 0, max = radar.max } = typeof m === 'string' ? { name: m } : m;

        return {
          id: name,
          label: name,
          scaleType: 'linear' as const,
          min,
          max,
        };
```

Not sure if there's any benefit in allowing that, but just wondering if we should use `Array<string | MetricConfig>` here.